molecule/default/converge.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  vars:
+    drupal_log_path: /var/log/vhosts/account/project/drupal.log
+    drupal_syslog_read_users:
+      - johndoe
+  pre_tasks:
+
+    - name: Update apt cache.
+      apt: update_cache=true cache_valid_time=600
```

Comment:
Apt tasks don't work on centos, 

https://github.com/AcroMedia/ansible-role-drupal-syslog/pull/1/checks?check_run_id=1443067720#step:4:381

Reply:
@dale-c-anderson : Fixed it. There were couple others as well.  Now works on all platforms.